### ISSUE TITLE
Stuck at checking for vaild IP 

### ISSUE THREAD

**Issue by PCGr385** [FriendsOfMDT/PSD#157]
I installed a fresh instance of PSD on my server and tested it a bunch and it was working perfect until all of a sudden PSD is not getting a valid IP after booting off WDS or ISO. Even on physical devices that I have tested with successfully before I am having the same issue. I have added the drivers for the network adapters and updated the boot files via "update deployment share" but the issue persists. In the command prompt with ipconfig I don't see any adapters and when I plug in an external adapter it shows up. It does work with external adapters but I'd really like to get the internal adapters working again.

Any ideas?

**Comment by PowerShellCrack** [comment]
this does still seem like a driver issue. Can you specify more details, like model, nic, drivers? Can you test on a VM jsut to make sure networking works?

**Comment by PCGr385** [comment]
Currently we are deploying to Lenovo t14 model: 20S0-0032US and Dell latitude 5410. I tested with a slightly different model of lenovo t14 20S1-S3L703 but that has stopped working as well. No drivers were changed between testing and when this issue started. The NICs on lenovo appear to be intel (10) I219-V and on dell is Intel (10) I219-LM. As for the drivers, I used the ones one the manufacturer's website extracted them with the built in extractor from the driver and import them into mdt.

It is working in a VM as well.

**Comment by PowerShellCrack** [comment]
Interesting that's it works with a VM. When u import the drivers in workbench; are you building a selection profile for Winpe and the boot media to use? Maybe try that, build a or folder, import only storage and nic drivers into that folder, and set it to install all drivers in winpe x64 settings. This will skip the check for driver type which should force all driver to install on new pe.

**Comment by PCGr385** [comment]
Yeah I have a selection profile for winpe with only the storage, nic and mouse/kb drivers with it set to install all drivers.

![image](https://github.com/FriendsOfMDT/PSD/assets/173704198/e7b5e8b3-5fde-4fb9-b24c-f69763808844)

**Comment by PCGr385** [comment]
I just tried injecting the drivers into the WIM file manually using DISM commands and it's now working on my devices internal NIC. So it seems like the drivers aren't getting injected into the WIM file when I update the deploymentshare.

**Comment by PowerShellCrack** [comment]
Must be an issue in the MDT workbench or selection profile. What is the type showing for drivers? Also check the path to ensure it is correct
